### ISSUE TITLE
Need to verify if a user is able to associate any random attachment to their post

### ISSUE THREAD

**Issue by vikasrohit** [topcoder-platform/tc-message-service#96]
This is to check with new attachments endpoints if there is way for a bad user to associate any random attachment to their post by replaying the topic/post creation call with `attachmentIds` containing some existing attachment id.

@gondzo do you think it is possible?

**Comment by vikasrohit** [comment]
fyi @maxceem

**Comment by gondzo** [comment]
I don't think so - we're only adding attachments that aren't already assigned to any post - https://github.com/appirio-tech/tc-message-service/blob/cc9e199fe7629fbfde81df32d68979cbd23ef963/app/models/posts.js#L270

Worst case scenario - if you call the service with attachment id that belongs to a different post, the attachment won't be added to the post, but you won't get an error either.

**Comment by vikasrohit** [comment]
Thanks @gondzo for the update.
> Worst case scenario - if you call the service with attachment id that belongs to a different post, the attachment won't be added to the post, but you won't get an error either.

Exactly this is the case I am concerned with. Which code is preventing that from happening?
